ALapina/grid_game
# Add Game Screens and Score

This MR contains 2 changes!

- First one is adding screens for game start, win and over
- The second one is having a working score that increasing on a good item

## How to test the changes

1. npm run dev to run the game
1. Check that there is now a title screen
1. And if you press ENTER - the game should start
1. When playing - game score is located in the top left corner
1. When you dig good stuff the score increases
1. If you dig bad stuff, the score decreasing
1. If your score super bad, you should see game over screen
